packages/create-miniapp-cli/template/config/_eslint.base.js

```
@@ -0,0 +1,45 @@
+const compat = new FlatCompat({
+  baseDirectory: import.meta.dirname,
+});
+
+export default tseslint.config(
+  {
+		ignores: ['.next']
+	},
+  ...compat.extends("next/core-web-vitals"),
+  {
+    files: ['**/*.ts', '**/*.tsx'],
+		extends: [
```

ESLint

[error] Expected indentation of 8 spaces but found 2 tabs.

(@typescript-eslint/indent)